bbc/simorgh
# WSTEAM1-722 - Remove media player placeholder from MediaArticle pages

Resolves JIRA [WSTEAM1-722](https://jira.dev.bbc.co.uk/browse/WSTEAM1-722)

Overall changes
======
Removes the placeholder on MediaArticle video player.

Code changes
======

- Logic to decide wether to show media player placeholder
- Opportunistic refactor of `ArticleMediaPlayer` tests
- Adds test case to check placeholder isn't in the media player

Testing
======
1. Visit http://localhost:7080/pidgin/articles/cvpde7nqj92o]
2. See no placeholder is rendered, and the iframe has rendered in straight away

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
